app/messaging/forward-message-request-to-sfd.js

```diff
@@ -0,0 +1,23 @@
+const { MessageSender } = require('ffc-messaging')
+const { sfdMessageTopic, sfdMessageRequestMsgType } = require('../config/index').messageQueueConfig
+
+const sendSfdMessageRequest = async (sfdMessageRequest) => {
+  const sender = new MessageSender(sfdMessageTopic)
+  const message = createMessage(sfdMessageRequest, sfdMessageRequestMsgType)
+
+  await sender.sendMessage(message)
+  await sender.closeConnection()
+}
+
+const createMessage = (body, type) => {
```

Comment:
I was expecting us to be matching the SFD contract as part of the scope of this ticket, but this is more of a dummy message